exercises/B-boolean-literals/exercise.js

```diff
@@ -6,6 +6,9 @@
 */
 
 let codeYourFutureIsGreat = true;
+let mozafarIsCool = false;
```

Comment:
All looks good, though try to use `const` instead of `let` in the first instance, in general reassignment of variables is confusing in large codebases

Reply:
Thank you, done.